### ISSUE TITLE
Bahamut gear upgrade request

### ISSUE THREAD

**Issue by Dartan82** [torilmud/issues#590]
https://docs.google.com/spreadsheets/d/1UEUomQ_mR0fzj-bexIv0dQrWcTrunVvB49GxxPoYOyc/edit#gid=0

- a silver ring with mirrored finish - change svsp to sf heal or add a sf heal or svsp buff to the csan proc

- platinum scalemail of the faithful - remove !evil flag

- a crown of the death dragon - damage/hp with a taunt proc... either make it a tank crown or a damage crown.  needs to be competitive with serpent circlet if it stays damage, otherwise add maxagi instead of damroll or just add hitroll

- the hollowed claw of a balor demon - remove ac, add HP
- an adamantium cloak   of elvenkind - remove mv, add hp or hitroll or maxdex or maxagi
- the crystallized tear of an astral deva - add a power to give spell focus to all types

- a stygian black dragon mask - add 5% all physicals to be competitive with drow mask
- a fiery red dragon mask - add warrior types.  why rogue only?

- a celestial longbow of carved yew wood - add keen, more damage enchants, or heal/blind, or debuffs

- the paws of the cheetah - just add haste instead of the power.  allows more flexible warrior builds instead of everyone wearing linked mail

-  a dragonhide component bag and haversack - give real stats.  dragonhide ammo belt has 1 dam vs. 25 mv vs. 5 agi.  agi should be maxagi, mv should be hp

- a crown of the north wind - remove the sv br condition or make it so the powers can actually land.  Change stats to damroll as it takes up a serpent slot for melee.  This comes from one of the hardest fights in the game but nobody wears it because it is junk

**Comment by jokkmeister** [comment]
I do not agree with Dartan on most instances here. I do not believe the items from Bahamut should be on par with higher end quest items. This is a zone where half of the group bots through it while if you are doing a quest you are actually doing a lot of work often over many boots so those items deserve to be better than something you just have to be lucky with dice to get.

I actually think Tia and Bahamut items need downgrades in many cases or the top end, hard to get quest items need upgrades like whark hoops, CM epic stuff. Lots of items in tough quests that never see any use because Tia (and probably soon Baha for some items) which I find very sad. Can just remove all quests and put those items in Tia and Baha and keep doing those zones over and over becuase it seems like the direction this place is taking. Too many good items in too few places.

There is a reason we see many new players all decked out in Tia gear before they have "regular" high end items and they almost never have top end quest items. They do Tia, Baha, ChP and the occasional trune maybe, but hell they can't even walk to Baldur's Gate from Silverymoon.

I'll give my view on the items listed in bold:

a silver ring with mirrored finish - change svsp to sf heal or add a sf heal or svsp buff to the csan proc - **I think there are already way too much sf_heal compared to other sf-items**

platinum scalemail of the faithful - remove !evil flag - **I like restrictions, it gives more options for other zone makers to make equally good items.**

a crown of the death dragon - damage/hp with a taunt proc... either make it a tank crown or a damage crown. needs to be competitive with serpent circlet if it stays damage, otherwise add maxagi instead of damroll or just add hitroll - **I agree make it more tanky instead.**

the hollowed claw of a balor demon - remove ac, add HP - **No need, still is used by everyone who has one and there are already plenty in game. Feels they are already more common than Frost Band.**

an adamantium cloak of elvenkind - remove mv, add hp or hitroll or maxdex or maxagi - **is good as it is (which is why those who have it uses it).**

the crystallized tear of an astral deva - add a power to give spell focus to all types

a stygian black dragon mask - add 5% all physicals to be competitive with drow mask - **Drow mask is so much harder to get it should be alot better.**

a fiery red dragon mask - add warrior types. why rogue only? - **Rogue only is good enough imo. They are the ones that need variation since everyone uses trickster. Warriors already have plenty of options.**

a celestial longbow of carved yew wood - add keen, more damage enchants, or heal/blind, or debuffs - **If this gets an upgrade then make sure Dragonhunter and, even more, Icicles bow get bumped to way better.**

the paws of the cheetah - just add haste instead of the power. allows more flexible warrior builds instead of everyone wearing linked mail - **No it's good as it is, does not need perm haste to make rangers even more twinkish than they are.**

 a dragonhide component bag and haversack - give real stats. dragonhide ammo belt has 1 dam vs. 25 mv vs. 5 agi. agi should be maxagi, mv should be hp -  **No these are containers and should not be OP. On par with trollide, decaying, silver quiver etc is more than good enough. I mean Avernus epic component bag got downgraded to useless and compare getting that to winning 4th bid on Baha.**


a crown of the north wind - remove the sv br condition or make it so the powers can actually land. Change stats to damroll as it takes up a serpent slot for melee. This comes from one of the hardest fights in the game but nobody wears it because it is junk  - **I agree powers should land when used (same goes for the caster orbs)**

**Comment by Dartan82** [comment]
1. Sunburst is already the best option and has no align restrictions..  it's 7 hp more without the 5% pierce.  Good only doesn't make sense when evil can also be faithful in their own right.  Based on your argument sunburst should be good only and scalemail should be evil then as that would have item diversity?
2. fiery dragon mask is the best rogue mask by far and makes warrior masks obsolete.
3. paws argument is stupid.  it's a rogue upgrade to make so they aren't forced to wield haste daggers...
4. so it's ok for melee to get their best stat on a quiver but not ok to have hp for casters or even maxint?
5. silver ring there is one option to get sf heal on ring.  
6. claw - yea and 20 hp isn't going to make or break going from band.  also band doesn't have align restrictions.

**Comment by jokkmeister** [comment]
When people stop going to Baha because the items suck then they might need upgrades. There are zones not being done for lack of good rewards, Baha is not one of them.

**Comment by Lilithelle** [comment]
a silver ring with mirrored finish - if anything maybe -5 ss, ss is important to a cleric for resisting silence.
PLATINUM scalemail of the FAITHFUL - its armor given to people that serve Bahamut...
crown of the death dragon - Yes, this needs something, no one would use over coiled, 6maxagi instead of 6dam.
hollowed claw of a balor demon - Lose 2 ac but gain fly almost on demand, seems worth it, rogue invariably lose something when they have to switch to fly eq.
adamantium cloak of elvenkind - was meant to be an goodable version of cloak of encasing darkness, which has 5% melee resists (except unarmed but that will be fixed), that's why this cloak has 4 dam and 50 moves but it should have the same melee resists as cloak of darkness then it will be fine.
crystallized tear of an astral deva - 20hp -5ss is solid, spell save is important, re: silence, pidibeple wears 2 spell save earrings (though they're sf-invo too).  Not every slot needs to be spell focus.
the paws of the cheetah - I agree should be full haste, but this isn't a bad compromise for the only non-wield rogue haste item in game.
crown of the north wind - Removing failable seems fair, the procs aren't over powered.  Its meant to be a hitter priest crown, that's why 7 hit.
a dragonhide component bag - I had it 5maxagi instead of 5agi, that was nixed, now it is worse than doeskin quiver I've solo'd 50 times, if 5maxagi is out (keep in mind tia quiver is pfa and 4maxdex) at least make it something useful like -5 spell save, para save, or breath save, that'd make it pro eq.
a celestial longbow of carved yew wood - It loads if blades don't, blades rock, this is mediocre at best.  If you're not fighting something evil its no better than several other bows, worse than magma bow.  Bow from Imix (Can be 2 personed) does more damage because it has fire proc).  At least add fire enchantment, adding brilliant would truly make it a worthy weapon.
a fiery red dragon mask - 3 masks are priest only, 1 is warrior only, 1 is rogue only; this is how I meant it to be, even if it disappoints terribly, not sure can we do something?  Realistically, no one would pick it over trickster, I wouldn't object to warriorable.
stygian black dragon mask - This disappoints me when compared to the other masks, it is inferior to randar's mask by 1 ac (though does have resists).  The other masks are 20hp (or 4 dam) and 8 sf or 8 maxstat, would it be so bad for this to be 20 hp 8maxagi?  It needs something I believe.  4 dam 8maxagi?  Would be a hitter mask then not tank mask, for tanking get randars or drow.  Plus, this is a major reward from zone, you only get it from killing Bahamut himself, and ties in with the quest that gets you started on Bahamut, from dragonspawn in Mir forest.
Lilithelle

**Comment by Lilithelle** [comment]
It is clearly much less rewarding than Tiamat, it is failable, if you don't have a good enough group you will not kill Bahamut, just failing once means killing 7 gold dragons which takes an hour and a pile of corpses.  Also, the full zone takes 6 hours with a good group, more with less.

**Comment by Lilithelle** [comment]
In 6 hours, with less than 15 people, and people that are just OK not stacked, I can do TK, ChP, Trune, and BC.  The fact that there are so many abstains when blade doesn't load, and that points are building up faster than in Tiamat, indicates there just aren't many items that are sought after, at least strongly.

**Comment by Dartan82** [comment]
adamantium cloak of elvenkind - A 1.5 hour zone vs. bahamut loot with fly isn't really that great.  
claw - There are fly potions too you know
ring - magma rings give the same stats so it is a small upgrade on top of it.  You need to wear the ring for 3 min to use yet it's not worth using over a healing ring...
component bag - why not maxwis or maxint?  maxagi might be seen as too good but there's tons of maxint and maxwis out there.
fiery dragon mask - you literally said you agreed to make it warriorable in tells but that one fix probably wouldn't make it but multiple would... lol
stygian - 4 dam 8 maxagi would be pretty good.  the priest masks are top tier and the only sf masks.  4 dam 8 maxagi would make is a much better grin I guess so not sure how people will take that.

**Comment by Dartan82** [comment]
To say "when people stop doing X zone" is an invalid comment.  There is literally one leader left and that person wants to hammer out Bahamut, Bahamut will be done.  So I guess IC vault needs upgrades because clearly Lilithelle doesn't like leading it.

**Comment by Lilithelle** [comment]
I never liked grin and never use it, ac0, terrible.

**Comment by jokkmeister** [comment]
Well people do this zone because they want to get the eq it gives, just as the eq is statted right now. No it's not Tiamat and maybe it should be treated like an ordinary zone in terms of bids since it has less items and not as good items. The solution is not to up the stats but rather to down the the time it takes to ddo. It's more time consuming than hard.

**Comment by Dartan82** [comment]
No they don't.  First round bids are only on very few items... 

If you want to get into difficulty, your epic quests are much easier to do for a guaranteed reward..

**Comment by jokkmeister** [comment]
How are they easier to do than type consent, follow and then enable triggers in nyybot?

I mean there is a reason many of the high end quests are not done often and it's because it takes alot of time and effort to do. Tiamat is being done (over done) so much because there are too many great rewards. Do we need another zone with the same issue? I don't think so. 

here are not many items in Baha I would say no to becuase they don't have good enough stats. The reason people don't bid and save points is that Blades are so amazing. There was the same issue with circlets in Tiamat too before they were common. Now they are single bids at times.

**Comment by Lilithelle** [comment]
Its not quite as simple as nyybot, i made it more difficult, like being able to flee 5 different directions from most fights requiring runs.  Stuff heals, limited time to do consort fights, if not enough people are doing runs it won't happen.  For patrols take downs are essential, if people aren't doing the job people will die.  Expecting no lazy afk followers probably won't happen.

**Comment by Dartan82** [comment]
High-end quests aren't done often simply because of the # of boots.  You're talking about something someone can choose to do within a month and is guaranteed a reward vs. something that takes 20+ to do to get a chance at something 4x a month.

To say nyybot can do Bahamut is false.  It was really easy to point out you specifically Adorel were afk and why you missed out on points in the first couple of runs of Bahamut because I docked you.

People aren't all abstaining for blade.  It's just what a few VB members are doing.  Again, something you don't see because you aren't actually at Bahamut.

**Comment by Dartan82** [comment]
Also per Lili's comment, when Gerin and I were leading with less afk people, we killed 2-3 patrols per run before wiping.  Still, much harder than Tia.

**Comment by jokkmeister** [comment]
Oh some of us may be able to finish a CM epic or equivalent within a month, but the vast majority can not and certainly not solo.

Yes I was docked points and have always been ok with that as well as open that I afk. I know of several people who play shaman or bard afk alot and is not docked points. So the penalizing afk is not really that fair imo.

Then why the complaints that people are abstaining bids because the other eq sucks?

**Comment by Dartan82** [comment]
smh not going to argue with your views.  you want a solo mud.

**Comment by jokkmeister** [comment]
No I want a mud that is not made up of 5-6 zones that people do over and over again. Please do not presume to tell me what I want.

**Comment by saerian** [comment]
I've done Bahamut a bit, and fully enjoy the zone. I like that it takes a lot of people, a lot of time, and that there's not enough gear for everyone (unlike Tiamat). That should make the equipment rarer, and more sought after. Right now though, it seems people only want the blades (as evidenced by all the dogpiles on them when they load). That said, I fully support the idea that some items should be changed to reflect that extra difficulty, time, and commitment from players to get it done. You can't do Bahamut with a Tiamat group.

I think Bahamut is a fantastic zone, and that there should be ever increasing more difficult zones added in the future. Maybe not as much time, but as player power increases, so should the zones that they do (progression).

**Comment by Nyyrazzilyss** [comment]
My opinion, if a zone is being completed there's no reason to upgrade anything that comes from it. The rewards would already be appropriate (or perhaps excessive and in need of downgrade?)

**Comment by Dartan82** [comment]
If Lilithelle leads, people will follow.  Nobody was doing the zone when Lilithelle couldn't do it.  So to your argument, the upgrades should have already occurred months ago.

**Comment by jokkmeister** [comment]
I think if the offensive powers on the items are made to land 100% instead of like 20-25% like now then that would be good, because powers are a poor mans proc and should land as effectively as a written proc imo.

**Comment by gonoph** [comment]
* mirrored ring - it needs something on par to the other ring items so it's not a niche item. I swap out the sf_heal rings when I need breath, or sv_spell for certain fights. that's a choice I make. right now, all mirrored has is a proc - which is questionable how good it is in a zone, and hps, which are marginal at best compared to the sv_bre or sv_spell I might need otherwise in that situation.
* scalemail - ok.. I get that it's for bahamut folks, when DT quest is fixed, most will be opting for upgraded sunburst instead, if you want baha scale to be "2nd best" or worn "when you can't quest sunburst upgrade" then I guess that's an ok place to be - except this is from one of the hardest if not the hardest zone in the mud.. and it's a vault item - so to be 2nd tier in that slot is kinda lame.
* hollowed claw - AC on rogue ring is kinda meh - hps are more important considering most rogues aren't tanking - or if they are, it won't be for very long *SPLAT*.
* elvenkind - not sure when I'd ever wear this compared to other stuff - I'm sure not putting points on a bid for a 1st or 2nd round bid... again, same argument as scalemail above.
* longbow - completely 3rd tier bow from one of the top end zones. That's kinda sad if you think about it - it's like a "parting gift" for when the swords don't load.. like "thank you for playing, here's your token T-Shirt!" Imix bow is better, and magma bow is better than that. longbow should be better than both.
* cheetah paws - for ranger, if you have tia armor, you're not going to use another haste item, and you're not going to swap out for something like blued breastplate except in certain circumstances. The argument that a perma haste foot item will make rangers more powerful is laughable. Rangers have tia armor - that's good enough. For fights were you swap out for another item, it's for specific reasons, and fights are shorter - that's what haste potions are for.
* cheetah paws - it makes sense for a rogue, as it's a PITA to carry around a bane or Rimi when you want some khanjari rage or healing, but it's also a PITA to have to keep engaging the power, and it breaks hide - unless your rogue is just pretending to be hitter-lite and roll with the hexblades - then breaking hide is a deal breaker. You know.. rogues are supposed to be sneaky.

**Comment by Lilithelle** [comment]
I compared 3 bows, bahamut, magma, fire plane.  I gave my ranger 35 hit and 48 damage in all cases, so it would measure only proc strength not hit/dam differences.  I use a training automaton with 30k hps.
magma - 580 seconds
fire plane - 580 seconds
bahamut vs evil aligned automaton - 556 seconds
bahamut vs non-evil automaton - 664 seconds
So bahamut bow is marginally better vs evil mobs, and 14% worse vs non-evil mobs, and the marginally better might just be luck with procs, not a real difference.

**Comment by Casorus** [comment]
Yep, bahamut gear sucks.  There's a few good items, but mostly not worth the effort.  Big disparity between Tiamat/Bahamut effort vs reward.

Like the orb of black dragonkind, it has amazing spell focus!  However the blind hardly EVER lands, and even if it does, it lasts a very short amount of time. 

 It's gotten to the point where I will not do bahamut in it's current state, not spending 5-8 hours of my time repeatedly dying, begging the bots to do runs, all so in the end I have a chance at some "meh" items.

  If perhaps it could be done with a smaller group, with people that are actually at keys, then maybe I would be open to it.  But by design, you need a TON of people to finish the zone, so the meager rewards are split up between the afkers and those who actually pay attention.

**Comment by Alurissi** [comment]
aside from banning or caging people so they can't go, i'd say just leave the rewards at the gruumsh approved podville level.  He rarely looks at difficulty or time, in doing things, and just wants everything to be level 1 newbie gear in stats.  Which is why you get one or two over powered ungodly eq, from a quest you can skip half of because of poor design, and stuff that's actually difficult, isn't worth doing.......

case in point, lets look at skullport:

archaic skull - ungodly stats, easy to get, and you can skip half the quest

headbands....same thing, skip half the quest, over statted rewards

skulls quest: terrible rewards, 8,000x harder than the 2 listed above.......

..........................

lets look at the last few years...............nothing is being tested, most all of annam's zones, quite a bit of lili's stuff, silvanus changes his zones years after they are in game......and when it comes to stats, it's just a i dont' like those stats, so i don't care how hard it is to get, or how easy......response from gruumsh.

Silvanus zones work, aren't terribly statted for time/work/reward........but again, poorly made, with a ton of things missing descriptions.  When he does get around to fixing issues that arise, he never tests them.......gem mephits, earth plane......if you don't know it, you haven't been raped for no good reason, in a place that houses a quest mob, that you can never quest due to his change to the side area, and good luck collecting xp tokens in there, as it was meant to be.......

Look at all the powers that came in with bahamut, that didn't work, at all.  A lot of those are from the areas gods, whom seem to have learned to just do something, not proof read it, and then never think of it again........the shevarash way.

In my honest opinion, most of bahamut's eq is a gross disappointment.  But, in a world where you can get a 6/6 perm haste proc dagger........well, you can if you are pidi, who seems to be the only person the quest is for......which puts both bane and lareth tooth to shame.  no reason it should be so high in stats, and the others so low, except for the fact, gruumsh didn't make it, and hasn't been allowed to touch it.  I say, roll a new char, put that eq on, and thank gruumsh for making everything = in stats to it.

why quest? 90% of the real quests are a joke when it comes to risk/reward.....speaking of which, good call letting lili give good align conquest eq, the same proc, but different stats than the original conquest eq...obviously another wonderful areas god decision not to look at what was being cloned...the quests are kinda dumb too, bring me 6 arms, here take these sleeves, bring me 9 boots, here take these boots.....bring me 4 hats, here take this hat..........if all we are doing is delivering a shipment to a shop, or reducing the number of each slot we carry around...., then what's the point?

.............................

basically, 
the zone should have never been implimented until it was tested.  hell, we had a pool to the astral sea load in a guest room instead of a guest last month.........something that is clearly a zone file error, and would have been caught in testing.

but yeah, never mind me, i'm not here, and you can thank beshaba and his 0 checks and balances from his boss, shevarash, for that.

Just love being banned for doing nothing except saying shevarash, police your admins, and fix your broken and already reported code.  I guess me and cherza have that in common, oh wait, no we don't, i didn't stalk anyone irl

**Comment by shevarash** [comment]
Just to be clear to @Alurissi and everyone else, Gerin has not been banned from Toril. I did raise the possibility to him, but that was intended to be both a last resort and a temporary measure. I have no desire to place permanent bans on any member of our community.

I apologize for derailing this ticket further, but I felt that this needed to be said.

**Comment by Dartan82** [comment]
So is this dead?

**Comment by jokkmeister** [comment]
Again, we still do the zone and people still want the eq, even though it is not uber, it doesn't have to be. But later it will probably need an item of volance or loop stature to make people who already have two of everything go. Called procs that recharge in bag is a must so we can get multiples. Otherwise when you have it all the reason to go is gone. Fun isn't enough.

**Comment by Dartan82** [comment]
Thanks Ikhar!  I highly value the opinion of someone who barely plays and doesn't really do the zone.

**Comment by jokkmeister** [comment]
I did it Sunday and there didn't seem to be a shortage of people wanting to do it. You were there as well. I rest my case.

**Comment by Dartan82** [comment]
15 man at start and we were about to abort.  20 at the end.  That is called a shortage.

**Comment by Dartan82** [comment]
This zone is fucking stupid and the imms don't give a shit.

**Comment by cmoncrief** [comment]
Hi guys,

GitHub is for reporting issues and helps us to prioritize them. I know tempers get heated and you're all passionate about this game, but please don't flame each other (or the staff) in here. Thanks!

As for this particular issue, it is still opena nd being worked. Thank you for your patience.

**Comment by Dartan82** [comment]
tired of waiting for the imms to get their heads out of their asses
bahamut loot is a fucking joke
but ikhar and marthammor's stupidass concept of how loot should be statted is utter garbage
ikhar's especially
if one of the 2 area gods is literally that blind on how gear should be started (marthammor) there's no point in playing anymore
any new content will be complete and utter garbage and not worth the effort
so we're just fucking playing tiamat every weekend and stacking volances and loops
